docker.rb

```diff
@@ -0,0 +1,99 @@
+cask "docker" do
```

Comment:
Never seen this file, but you can add it to `.gitignore` so it won't be uploaded

Reply:
This is still here